tsconfig.json

```diff
@@ -1,25 +1,31 @@
 {
   "compilerOptions": {
-    "target": "es6",
-    "lib": [
-      "es2017"
-    ],
+    // Basic Options
+    "target": "ES2017",
```

Comment:
ES2017 is fully supported in all web browsers, and in all Node versions >= 9.
https://node.green/#ES2017

If we want to maintain support for Node 8, I'm ok with bringing down the target, but since Node 10 is the current older LTS version, and Node 8 end of life'd on December 31st, I think this is a safe change.

Reply:
ripple-lib supports node 8

https://github.com/ripple/ripple-lib/blob/develop/tsconfig-base.json#L4

https://github.com/ripple/ripple-lib/blob/develop/.travis.yml#L3

Not sure there's much benefit(?) to targeting ES2017, would lean toward es6 for now and it's easy to bump it up later